pkg/syncer/resource_syncer.go

```diff
@@ -61,16 +61,18 @@ const (
 )
 
 func (d SyncDirection) String() string {
+	s := ""
```

Comment:
The behaviour is different,

```suggestion
	s := "unknown"
```

would preserve the coverage improvement and keep the existing behaviour. I know it’s rather theoretical, but a `SyncDirection` can take any `int` value...

Reply:
yeah it shouldn't happen  - "unknown" isn't actually an enum so perhaps "" is more appropriate but either way...I changed it back.